tests/post_training/test_quantize_conformance.py

```diff
@@ -0,0 +1,334 @@
+import os
+import re
+import logging
+from pathlib import Path
+import numpy as np
+
+import torch
+import nncf
+import onnx
+from torchvision import datasets
+from torchvision import transforms
+from torchvision.transforms import InterpolationMode
+
+import pytest
+
+import timm
+from tqdm import tqdm
+from sklearn.metrics import accuracy_score
+
+import openvino.runtime as ov
+
+from model_scope import get_validation_scope
+import copy
+
+NOT_AVAILABLE_MESSAGE = "N/A"
+DEFAULT_VAL_THREADS = 4
+
+
+def create_timm_model(name):
+    model = timm.create_model(
+        name, num_classes=1000, in_chans=3, pretrained=True, checkpoint_path=""
+    )
+    return model
+
+
+def get_model_transform(model):
+    config = model.default_cfg
+    normalize = transforms.Normalize(mean=config["mean"], std=config["std"])
+    input_size = config["input_size"]
+    resize_size = tuple(int(x / config["crop_pct"]) for x in input_size[-2:])
+
+    RESIZE_MODE_MAP = {
+        "bilinear": InterpolationMode.BILINEAR,
+        "bicubic": InterpolationMode.BICUBIC,
+        "nearest": InterpolationMode.NEAREST,
+    }
+
+    transform = transforms.Compose(
+        [
+            transforms.Resize(
+                resize_size, interpolation=RESIZE_MODE_MAP[config["interpolation"]]
+            ),
+            transforms.CenterCrop(input_size[-2:]),
+            transforms.ToTensor(),
+            normalize,
+        ]
+    )
+
+    return transform
+
+
+def get_torch_dataloader(folder, transform, batch_size=1):
+    val_dataset = datasets.ImageFolder(root=folder, transform=transform)
+    val_loader = torch.utils.data.DataLoader(
+        val_dataset, batch_size=batch_size, num_workers=2, shuffle=False
+    )
+    return val_loader
+
+
+def export_to_onnx(model, save_path, data_sample):
+    torch.onnx.export(
+        model,
+        data_sample,
+        save_path,
+        export_params=True,
+        opset_version=13,
+        do_constant_folding=False,
+    )
+
+
+def export_to_ir(model_path, save_path, model_name):
+    command_line = f"mo -m {model_path} -o {save_path} -n {model_name}"
+    os.popen(command_line).read()
+
+
+def run_benchmark(model_path):
+    command_line = f"benchmark_app -m {model_path} -d CPU -niter 300"
+    cmd_output = os.popen(command_line).read()
```

Comment:
We have `tests.common.command.Command` for that. Same for all other spots where a command has to be executed in the shell.

Reply:
There is no type hint, help, need to understand how it works but valid. Reworked.